sc3nb/sc.py

```diff
@@ -598,30 +618,36 @@ def __read_loop_unix(self, output, queue):
                     queue.put(out)
                     if self.console_logging:
                         # to remove ansi chars
-                        out = ansi_escape.sub('', out.decode())
+                        out = ANSI_ESCAPE.sub('', out.decode())
                         # print to jupyter console...
                         os.write(1, out.encode())
             except EOFError:
                 pass
             time.sleep(0.001)
 
-    def Buffer(self, **kwargs):
+    def Buffer(self, *args, **kwargs):
         """
         Documentation see: :class:`Buffer`
         """
-        return Buffer(self, **kwargs)
+        return Buffer(self, *args, **kwargs)
 
-    def Synth(self, **kwargs):
+    def Synth(self, *args, **kwargs):
         """
         Documentation see: :class:`Synth`
         """
-        return Synth(self, **kwargs)
+        return Synth(self, *args, **kwargs)
 
-    def SynthDef(self, **kwargs):
+    def SynthDef(self, *args, **kwargs):
         """
         Documentation see: :class:`SynthDef`
         """
-        return SynthDef(self, **kwargs)
+        return SynthDef(self, *args, **kwargs)
+
+    def SynthFamily(self, *args, **kwargs):
```

Comment:
We need to  discuss SynthFamily - I suggest to solve this differently for now

Reply:
I removed SynthFamily for now. If we feel the need to add something like this we can add it in another PR.